test/system/post_test.rb

```diff
@@ -0,0 +1,32 @@
+require "application_system_test_case"
+# https://guides.rubyonrails.org/testing.html#implementing-a-system-test
+
+class PostTest < ApplicationSystemTestCase
+
+  def setup
+    activate_authlogic
+  end
+
+  test 'posting from the editor' do
+    visit '/'
```

Comment:
Ah, i bet it's choking on `2019-04-26 22:08:19 +0000: Rack app error handling request { GET /url/to/image.png }` here! On the front page will be the "scraped image"!